tools/protovalidate-conformance/main.go

```diff
@@ -62,7 +63,14 @@ func main() {
 		res.Fdset = req.Fdset
 		resultSet.AddSuite(res, cfg.verbose)
 		return nil
-	})
+	}
+
+	var err error
+	if cfg.benchmark > 0 {
+		err = suites.NewBenchmark(cfg.benchmark, cases.KitchenSink()).Run(cfg.suiteFilter, exec)
```

Comment:
Hmm, this looks odd. Why not use the suiteFilter to select the suites to benchmark, instead of just using kitchen sink? Is that really supposed to be a suiteFilter and not a caseFilter?